### ISSUE TITLE
Bad gateway

### ISSUE THREAD

**Issue by Pogozhelskaya** [JetBrains-Research/CFPQ_Data#43]
Link to download [**Geospecies**](https://old.datahub.io/dataset/geospecies) doesn't work in README.

**Comment by gsvgit** [comment]
Looks like datahub.io is broken. I'm not sure that there is an another way to get this file, but I try to find it.

**Comment by gsvgit** [comment]
@Pogozhelskaya Try this link: http://rdf.geospecies.org/geospecies.rdf.gz

**Comment by Pogozhelskaya** [comment]
> @Pogozhelskaya Try this link: http://rdf.geospecies.org/geospecies.rdf.gz

Thanks! Link works, but it seems that data is corrupted. I've tried to import this RDF to Neo4j, but got parse error:
![image](https://user-images.githubusercontent.com/31728695/109031208-8f58b880-76d5-11eb-9b0b-b26f2185af24.png)

Also, I downloaded Geospecies from CFPQ_Data (logs are below), however it has the same problem.
![image](https://user-images.githubusercontent.com/31728695/109032774-0a6e9e80-76d7-11eb-8ea7-c194efe362ee.png)
